sdk/keyvault/.dict.txt

```diff
@@ -1,2 +1,17 @@
+cbcpad
+ciphertext
```

Comment:
Tip: this an "plaintext" (though seems it wasn't a problem) could go in `.vscode/cspell.json`. They are general enough. It's fine here too, though.